bindings/azure/blobstorage/blobstorage.go

```diff
@@ -22,16 +23,32 @@ import (
 
 const (
 	blobName                 = "blobName"
-	contentType              = "ContentType"
-	contentMD5               = "ContentMD5"
-	contentEncoding          = "ContentEncoding"
-	contentLanguage          = "ContentLanguage"
-	contentDisposition       = "ContentDisposition"
-	cacheControl             = "CacheControl"
-	deleteSnapshotOptions    = "DeleteSnapshotOptions"
+	marker                   = "marker"
```

Comment:
a constant that just repeats the content is not so useful. Please, add more context in the name for improved readability. This feedback is for the other constants too. I know it was already like that but this is an opportunity for us to raise the bar here.

Reply:
Do you have an example?

Reply:
I have added a prefix `metadataKey` for each constant. Hope this is OK?

Reply:
I think it would be great to add comment where define them and those meanings.

Reply:
@daixiang0 Do you mean for each or only for the group?

Reply:
@daixiang0 Just to be sure. You suggest to add a comment for each constant where the meaning is described?

```go
const (
	// Defines the name of the blob
	metadataKeyBlobName = "blobName"
	// Defines a string value that identifies the portion of the list to be returned with the next list operation.
	// The operation returns a marker value within the response body if the list returned was not complete. The marker
	// value may then be used in a subsequent call to request the next set of list items.
	metadataKeyMarker             = "marker"
        ...
)
```

Reply:
Is there any doc for those?

>  Defines the name of the blob
Comments like this make no sense.

Reply:
I have also updated the doc: https://github.com/dapr/docs/pull/1623/files

Please don't get me wrong. But could you please be more precise with the changes you suggest? I think otherwise we are going around in circles here

Reply:
Is there a doc for "blobName", "marker" and so on? Add link here. If not, add comment for each if necessary.

Reply:
Is this OK now?